PurchasesCoreSwiftTests/Mocks/MockSKProduct.swift

```diff
@@ -40,7 +40,7 @@ class MockSKProduct: SKProduct {
     @available(iOS 11.2, tvOS 11.2, macOS 10.13.2, *)
     lazy var mockDiscount: SKProductDiscount? = nil
 
-    @available(iOS 11.2, tvOS 11.2, macOS 10.13.2, *)
+    @available(iOS 12.2, tvOS 12.2, macOS 10.13.2, *)
```

Comment:
this is the main change related to the failed tests. The problem is that although [`SKProductDiscount`](https://developer.apple.com/documentation/storekit/skproductdiscount) is available from **iOS 11.2**, the property [`discounts`](https://developer.apple.com/documentation/storekit/skproduct/3043534-discounts) is available from **iOS 12.2** 😓 So, we have to update the availability of this property and the derived tests.

Reply:
Great catch!